src/main/java/com/dnastack/ga4gh/tables/cli/output/publish/GCSPublisher.java

```diff
@@ -57,7 +57,7 @@ public void publish(Table table) {
     public void publish(ListTableResponse table) {
         String tableInfoJson = format(table);
         String root = GcsUtil.getObjectRoot(destination);
-        String tableInfoPage = root == null ? "tables" : root + "/tables";
+        String tableInfoPage = this.destination + "/tables";
```

Comment:
Sanity check: Does this still function if you have trailing slash in your destination? (i.e. does it break by going foo.bar/baz//tables ?)

Reply:
I'm not sure if it would break, but I added a small function to prevent this

Reply:
Better: Test it, if it does break, write a test that checks that it's handling the // properly. Now that your test fails, add your small func that protects.

If it doesn't fail, then it's being correctly handled and you don't need to manually enforce (there's a really good chance the networking code handles this for free - like your browser probably does: https://github.com//dnastack should resolve in your browser).